src/calculateRentalCost.js

```diff
@@ -29,6 +29,19 @@
 
 function calculateRentalCost(days) {
   // write code here
+  const costPerDay = 40;
+  const daysForBigDiscout = 7;
+  const daysForSmallDiscout = 3;
+  const smallDiscount = 20;
+  const bigDiscount = 50;
+
+  if (days >= daysForBigDiscout) {
+    return (days * costPerDay) - bigDiscount;
```

Comment:
We are using `days * costPerDay` multiple times, maybe we can move it to a const?